cdi/pom.xml

```diff
@@ -167,12 +166,13 @@
                 <artifactId>maven-bundle-plugin</artifactId>
                 <configuration>
                     <instructions>
-                        <Fragment-Host>org.hibernate.validator</Fragment-Host>
+                        <Bundle-SymbolicName>${hibernate-validator-cdi.module-name}</Bundle-SymbolicName>
+                        <Fragment-Host>${hibernate-validator.module-name}</Fragment-Host>
                         <Import-Package>
-                            org.hibernate.validator.*;version="[${project.version},${project.version}]",
```

Comment:
Note that this was done on purpose, but reading [this explanation](https://felix.apache.org/documentation/tutorials-examples-and-presentations/apache-felix-osgi-faq.html#should-a-bundle-import-its-own-exported-packages) again, it seems ok to not do it in our case.